README.md

```diff
@@ -234,7 +234,6 @@ An example of using `@fastify/swagger` with `static` mode enabled can be found [
  | ---------------- | --------  | ----------------------------------------------------------------------------------------------------------------------------- |
  | hiddenTag        | X-HIDDEN  | Tag to control hiding of routes.                                                                                              |
  | hideUntagged     | false     | If `true` remove routes without tags from resulting Swagger/OpenAPI schema file.                                              |
- | initOAuth        | {}        | Configuration options for [Swagger UI initOAuth](https://swagger.io/docs/open-source-tools/swagger-ui/usage/oauth2/).         |
```

Comment:
Good spot on this one, it needs removing from the migration document as well.

Reply:
The migration document is actually right?

Reply:
Misread, ignore me.

Reply:
I would never ignore you ;)